transport/rif_comms/node.py

```diff
@@ -83,16 +83,17 @@ def _receive_messages(self):
         Iterate over the Notification stream and block thread to receive messages.
         """
         for notification in self._our_topic_stream:
-            raiden_message = self._notification_to_message(notification.channelNewData)
-            if raiden_message:
-                self.log.info("incoming message", message=raiden_message)
-                self._handle_message(raiden_message)
+            raiden_messages = self._notification_to_messages(notification.channelNewData)
+            if raiden_messages:
+                self.log.info("incoming messages", messages=raiden_messages)
+                for raiden_message in raiden_messages:
+                    self._handle_message(raiden_message)
```

Comment:
```suggestion
            for raiden_message in self._notification_to_messages(notification.channelNewData):
                self.log.info("incoming message", message=raiden_messages)
                self._handle_message(raiden_message)
```

Reply:
returning `[]` (instead of `None`) on `_notification_to_messages` will do away with the problem of attempting to iterate on `NoneType`